main.cpp

```diff
@@ -225,6 +234,11 @@ int CdDrive::findSpeed()
     return speed;
 }
 
+void CdDrive::maxSpeedCout()
```

Comment:
Name things what they are. 
Name functions based on what they do, and make sure that the name includes a verb
- [ ] revise